lib/includes.js

```diff
@@ -0,0 +1,189 @@
+var Promise = require('bluebird'),
+    _ = require('lodash'),
+    $http = require('http-as-promised');
+
+module.exports = function (adapter, schemas) {
+
+    function appendLinkedResources(body, primaryResourceName, inclusions) {
+
+        var primarySchema = schemas[primaryResourceName];
+
+        function isRemote(branch, key) {
+            return null != branch && null != branch.refs && null != branch.refs[key] && null != branch.refs[key].def && null != branch.refs[key].def.baseUri;
```

Comment:
That line is checking only single thing, if the branch object has properly nested `baseUri` property, which is the only thing needed to check if branch `isRemote`.